experimental/FTheoryTools/src/G4Fluxes/special_attributes.jl

```diff
@@ -225,19 +226,21 @@ function well_quantized_ambient_space_models_of_g4_fluxes(m::AbstractFTheoryMode
 
 
   # (3) Are intersection numbers known?
-  inter_dict = Dict{NTuple{4, Int64}, ZZRingElem}()
-  s_inter_dict = Dict{String, ZZRingElem}()
-  if has_attribute(m, :inter_dict)
-    inter_dict = get_attribute(m, :inter_dict)
-  end
-  if has_attribute(m, :s_inter_dict)
-    s_inter_dict = get_attribute(m, :s_inter_dict)
-  end
+  inter_dict = if has_attribute(m, :inter_dict)
+    get_attribute(m, :inter_dict)
+  else
+    Dict{NTuple{4, Int64}, ZZRingElem}()
+  end::Dict{NTuple{4, Int64}, ZZRingElem}
```

Comment:
If the attribute is present, this code ends up scanning the dictionary twice. You can avoid it like this:
```suggestion
  inter_dict = get_attribute(m, :inter_dict) do
    Dict{NTuple{4, Int64}, ZZRingElem}()
  end::Dict{NTuple{4, Int64}, ZZRingElem}
```

Reply:
Thank you. Will update this shortly.